### PR TITLE
Start review of managing accounts with Java

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

File: java/manage-accounts.md [hunk 1]
```diff
@@ -76,29 +67,28 @@ BigInteger nonce =  ethGetTransactionCount.getTransactionCount();
...
 
-The first form of wallet is the JSON encryted keystore, it is a password-encrypted version of the private key. This is the most standard way used by clients such as [Pantheon](https://pegasys.tech/) or [Geth](https://geth.ethereum.org/) but also by Online tool like [MyEtherWallet](https://www.myetherwallet.com/) to secure a private key from potential attackers.
+The first form of wallet is the JSON encryted keystore, which is a password-encrypted version of the private key. This is the most standard way used by clients such as [Pantheon](https://pegasys.tech/) or [Geth](https://geth.ethereum.org/), but also by online tools like [MyEtherWallet](https://www.myetherwallet.com/) to secure a private key from potential attackers.
 
 Web3j provides a utility class called `WalletUtils` to load a wallet into a `Credentials` object (wrapper containing the account address and the keypair).
 
+<!-- TODO: Where do these three details come from? -->
+
 ```java
 String walletPassword = "secr3t";
```
Comment (ChrisChinchilla) [inline java/manage-accounts.md]: @gjeanmart Could we add here where someone might find these details?

File: java/manage-accounts.md [hunk 2]
```diff
@@ -136,7 +119,7 @@ Credentials credentials = WalletUtils.loadBip39Credentials(password, mnemonic);
 
 ![](https://imgur.com/xN2Ruaj.png)
 
-By default, Web3j uses a derivation path equals to `m/44'/60'/0'/1`. However, it is possible to open another account on a different path:
+By default, Web3j uses a derivation path equal to `m/44'/60'/0'/1`. However, it is possible to open another account on a different path:
```
Comment (ChrisChinchilla) [inline java/manage-accounts.md]: @gjeanmart As a beginners post, this might need explanation.

Comment (gjeanmart) [reply java/manage-accounts.md]: I added a link to [this article](https://medium.com/myetherwallet/hd-wallets-and-derivation-paths-explained-865a643c7bf2) from MEW which explain this complex but very important concept of Mnemonic derivation path!
Let me know if you think it is enough to explain and understand this.

**Comment by gjeanmart** [comment]
Thanks for the review @ChrisChinchilla !

**Pull request merged**